service/orchestrator/metrics.json

```diff
@@ -390,6 +446,45 @@
       }
     }
   },
+  {
+    "id": "TlsCipherSuites",
```

Comment:
We should probably rename them to `TransportEncryptionXXX` in the future to harmonize them with the others, but I guess for now its ok.